Makefile.pre.in

```diff
@@ -1730,13 +1730,17 @@ tags::
 	ctags -w $(srcdir)/Include/*.h $(srcdir)/Include/cpython/*.h $(srcdir)/Include/internal/*.h
 	for i in $(SRCDIRS); do ctags -f tags -w -a $(srcdir)/$$i/*.[ch]; done
 	ctags -f tags -w -a $(srcdir)/Modules/_ctypes/*.[ch]
+	find $(srcdir)/Lib -type f -name "*.py" -not -name "test_*.py" -not -path "*/test/*" | ctags -f tags -w -a -L -
 	LC_ALL=C sort -o tags tags
 
 # Create a tags file for GNU Emacs
 TAGS::
+	touch tags
```

Comment:
Why do you need this line? If I invoke the makefile from another place won't this create a 'tags' file in my current working directory?

Reply:
`etags -a` appends to the current tags, (regardless of whats in it), if you run `make tags` and `make TAGS` you'll end up with a corrupted file.
the `make tags` commands will find the first pass without the `-a` append, so it resets the file. I'll update this to run in a different order

Reply:
Updated in https://github.com/python/cpython/pull/17721/commits/7e31b49c40bf969418a79a09b85fb12071a3356f